Makefile

```diff
@@ -142,6 +142,15 @@ EXTRA_LD_FLAGS  :=
 #
 # Default Tool options - can be overridden in {mcu}.mk files.
 #
+DEBUG_MIXED = no
+
+ifeq ($(DEBUG),INFO)
```

Comment:
You don't need these as you would provide the DEBUG_MIXED=yes on the command line like DEBUG=GDB

Reply:
I'd rather just have debug and info builds have the correct behaviour, so I didn't intend setting DEBUG_MIXED from the command line. The concern expressed above was over accidentally including debug objects in a full build meant for flight, which this addressed. INFO and GDB options are explicitly for debug for which a mixed build is the norm.